### PR TITLE
Add local stats for augment combinations

### LINKED ISSUE #612: Actually update combination stats

### PR DESCRIPTION
**Pull request by toberge**
Closes #612

### CODE REVIEW

**Comment by toberge** [comment]
prevent users from quitting a match early and saving weird data

**Comment by toberge** [comment]
> prevent users from quitting a match early and saving weird data

Won't do this now as it requires further work on the stat-storing code, which shouldn't be prioritized right now.

File: Assets/Scripts/Gamestate/Round.cs [hunk 1]
```diff
@@ -119,7 +119,7 @@ private void CheckWinCondition(PlayerManager lastKiller)
     {
         if (livingPlayers.Count < 2)
         {
-            winner = livingPlayers.FirstOrDefault()?.identity;
+            winner = lastKiller?.identity;
```
Comment (Fueredoriku) [inline Assets/Scripts/Gamestate/Round.cs]: Last living player is not necessarily last killer, is this change intended to change the logic or was it a refactor with unintended consequences?

**Review by Fueredoriku — CHANGES_REQUESTED** [review]
Nice, refactors and changes 😁👍
 There is just one change I think may lead to unintended consequences which I think might need to be adressed before merge

File: Assets/Scripts/Gamestate/Round.cs (hunk 1 above)
Comment (toberge) [reply Assets/Scripts/Gamestate/Round.cs]: Was a refactor, the method's parameter was unused and this change technically resolved that. Reverted and removed the parameter.

**Review by Fueredoriku — APPROVED** [review]
Very nice, will be great for the improved future gallery 🤠
